modules/serverless-rn-1-11-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-11-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.11.0
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+* Knative Eventing on {ServerlessProductName} is now GA.
```

Comment:
Looks like elsewhere in the Serverless release notes it's spelled out "Generally Available (GA)", if you wanna match that here, too.